pkg/scsi/sbc.go

```diff
@@ -54,7 +54,8 @@ func (sbc SBCSCSIDeviceProtocol) InitLu(lu *api.SCSILu) error {
 	lu.Attrs.SWP = false
 	lu.Attrs.SenseFormat = false
 	lu.Attrs.VendorID = "GOSTOR"
-	lu.Attrs.ProductID = "VIRTUAL-DISK"
+	lu.Attrs.ProductID = "GOTGT-VDISK"
+	lu.Attrs.ProductRev = "1.0"
```

Comment:
According to SPC-4, the product version should be 4 bytes ASCII code.
I would like to have another version code for this and add this into pkg/version.
How about DiskVersion? Or we can just follow the version rule of SPC-4.